src/main/java/org/cryptomator/frontend/fuse/ReadOnlyAdapter.java

```diff
@@ -70,10 +75,12 @@ public ReadOnlyAdapter(@Named("root") Path root, @Named("maxFileNameLength") int
 		this.linkHandler = linkHandler;
 		this.attrUtil = attrUtil;
 		this.hasOpenFiles = () -> fileFactory.getOpenFileCount() != 0;
+		this.toFuseEncoder = toFuseEncoder;
 	}
 
 	protected Path resolvePath(String absolutePath) {
-		String relativePath = CharMatcher.is('/').trimLeadingFrom(absolutePath);
+		var normalizedAbsolutePath = Normalizer.normalize(absolutePath, Normalizer.Form.NFC);
```

Comment:
In order to replace iconv, we don't want hard-coded UTF8-NFC. Instead both ends should be configurable, as discussed in #56

Reply:
furthermore, resolvePath is no only called when filenames are passed from nio→fuse, but also fuse→nio. In the latter case, we need reverse transcoding.